package.json

```diff
@@ -69,17 +70,11 @@
 		"ts-node": "^10.9.1",
 		"typescript": "^4.7.4",
 		"web3": "4.0.1-rc.2",
-		"web3-core": "4.0.1-rc.2",
-		"web3-eth": "4.0.1-rc.2",
 		"webpack": "^5.75.0",
 		"webpack-cli": "^5.0.1"
 	},
 	"peerDependencies": {
-		"web3-core": ">= 4.0.1-rc.2 < 5",
-		"web3-eth": ">= 4.0.1-rc.2 < 5",
-		"web3-eth-abi": ">= 4.0.1-rc.2 < 5",
-		"web3-eth-contract": ">= 4.0.1-rc.2 < 5",
-		"web3-types": ">= 1.0.0-rc.2 < 5",
-		"web3-validator": ">= 1.0.0-rc.2 < 5"
+		"web3": ">= 4.0.1-rc.2 < 5",
+		"web3-core": ">= 4.0.1-rc.2 < 5"
```

Comment:
Now we can make all dependencies to main 4x release instead of RC.

Reply:
Done